TARGET_DNABC = expand(PROJECT_DIR + "/demultiplexed_fastq_gz/{sample}_{direction}.fastq.gz", sample = SAMPLE_IDS, direction = ["R1", "R2"])

TARGET_QIIME_IMPORT = [str(QIIME_OUTPUT_DIR + "/demux.qzv")]

TARGET_DEMUX_STAT = [str(QIIME_OUTPUT_DIR + "/demux_stat/per-sample-fastq-counts.csv")]

TARGET_DENOISE = [str(DENOISE_DIR + "/feature_table/feature-table.tsv")]

TARGET_REPSEQ = [str(DENOISE_DIR + "/representative_seq_fasta/dna-sequences.fasta")]

TARGET_TAXONOMY = [str(DENOISE_DIR + "/taxonomy/taxonomy.tsv")]

TARGET_TREE = [str(DENOISE_DIR + "/rooted-tree.qza")]

TARGET_DIVERSITY = [str(CORE_METRIC_DIR + "/faith_pd.tsv"),
                    str(CORE_METRIC_DIR + "/uu.tsv"),
                    str(CORE_METRIC_DIR + "/wu.tsv"),
		    str(CORE_METRIC_UNRAREFIED_DIR + "/faith_pd_unrarefied.tsv"),
		    str(CORE_METRIC_UNRAREFIED_DIR + "/uu_unrarefied.tsv"),
		    str(CORE_METRIC_UNRAREFIED_DIR + "/wu_unrarefied.tsv")]

TARGET_UNASSIGN = [str(DENOISE_DIR + "/representative_seq_fasta/unassigned/unassigner_output.tsv")]

TARGET_DADA2SP = [str(DENOISE_DIR + "/representative_seq_fasta/dada2/dada2_species_assignments.tsv"),
                  str(DENOISE_DIR + "/representative_seq_fasta/dada2/dada2.RData")]

TARGET_VSEARCH = [
    str(DENOISE_DIR + "/representative_seq_fasta/vsearch/vsearch_report.tsv"),
    str(DENOISE_DIR + "/representative_seq_fasta/vsearch/vsearch.fasta")
    ]

TARGET_ALL = (
    TARGET_DNABC +
    TARGET_QIIME_IMPORT +
    TARGET_DEMUX_STAT +
    TARGET_DENOISE +
    TARGET_REPSEQ +
    TARGET_TAXONOMY +
    TARGET_TREE +
    TARGET_DIVERSITY +
    TARGET_UNASSIGN #+
    #TARGET_DADA2SP +
    #TARGET_VSEARCH
)
